teku/src/main/java/tech/pegasys/teku/cli/options/Eth2NetworkOptions.java

```diff
@@ -49,6 +49,16 @@ public class Eth2NetworkOptions {
       arity = "1")
   private String initialState;
 
+  @Option(
+      names = {"--ignore-weak-subjectivity-period-enabled"},
+      paramLabel = "<BOOLEAN>",
+      description = "Foo.",
+      arity = "0..1",
+      fallbackValue = "true",
+      defaultValue = "false",
```

Comment:
fairly sure we dont generally set the default value in a boolean, just a fallback, and the boolean default is in the variable initialisation...